tests/testthat/test_holiday.R

```diff
@@ -110,9 +110,16 @@ test_that("POSIXct class", {
     holiday_ind$day[is_equal_1(holiday_ind$day_Easter)],
     exp_dates$date[exp_dates$holiday == "Easter"]
   )
```

Comment:
The real problem is the `as.POSIXct()` converters. I think you could insulate yourself from R-devel a little more by using lubridate converters.

i.e. use this at the top of the test instead

```r
test_data$day <- lubridate::as_datetime(test_data$day, tz = "UTC")
exp_dates$date <- lubridate::as_datetime(exp_dates$date, tz = "UTC")
```

and this in the test:

```r
lubridate::as_datetime(NA, tz = "UTC")
```

Those converter functions still aren't perfect but I think they should work here

Reply:
Done! Thank you!